gp-templates/translations.php

```diff
@@ -309,8 +309,12 @@
 	</form>
 </div>
 
-<table id="translations" class="translations clear<?php if ( 'rtl' == $locale->text_direction ) {
-	echo ' translation-sets-rtl'; } ?>">
+<table id="translations" class="translations clear
+<?php
+if ( 'rtl' == $locale->text_direction ) {
+	echo ' translation-sets-rtl'; }
+	?>
```

Comment:
This should also be changed to only echo a variable, see the sub-project example.

Reply:
As it's not the only class, I'll call the variable `$class_rtl`.
Also, I think we should start escaping things, I'll use `esc_attr()`.